services/QuillLMS/client/app/bundles/Shared/utils/constants.ts

```diff
@@ -2,3 +2,7 @@ export const DEFAULT_HIGHLIGHT_PROMPT = "As you read, highlight two sentences "
 export const BECAUSE = 'because';
 export const BUT = 'but';
 export const SO = 'so';
+export const READ_PASSAGE_STEP = 1
+export const BECAUSE_PASSAGE_STEP = 2
+export const BUT_PASSAGE_STEP = 3
+export const SO_PASSAGE_STEP = 4
```

Comment:
I might call these constants `READ_PASSAGE_STEP_NUMBER` etc to be fully descriptive.